cuegui/cuegui/FilterDialog.py

```diff
@@ -447,7 +447,7 @@ def createAction(self):
                         "How much memory (in GB) should each render layer require?",
                         4.0,
                         0.1,
-                        47.0,
+                        251.0,
```

Comment:
Let's move this number to a constant at the top of the file and use that constant everywhere instead.

Reply:
Sure thing. Sorry about that, this passed unchecked. Will get back to my team to avoid such type of quality issues.

Reply:
No worries, that's what we're here for :)